### ISSUE TITLE
On windows if the adaptor name is in Chinese, result.interface failed to parse unicode adaptor name correctly.

### ISSUE THREAD

**Issue by WONDERlust91** [silverwind/default-gateway#14]
version 5.0.0

When I use Chinese on adaptor name.

![image](https://user-images.githubusercontent.com/23548132/59521561-25977b80-8eff-11e9-883b-950f35f1e67d.png)

![image](https://user-images.githubusercontent.com/23548132/59521669-6abbad80-8eff-11e9-9297-2c5e08bba2d6.png)

I changed the adaptor name in English.

![image](https://user-images.githubusercontent.com/23548132/59521736-9b034c00-8eff-11e9-8478-93d04a2d07d6.png)

![image](https://user-images.githubusercontent.com/23548132/59521766-a8b8d180-8eff-11e9-8b03-758dc83168b3.png)

**Comment by silverwind** [comment]
Does not look like there is an easy fix. In my tests I only get a bunch of question marks:

````
$ wmic path Win32_NetworkAdapter get Index,NetConnectionID /format:table | grep 12 | hexdump -C | head -1
00000000  31 32 20 20 20 20 20 3f  3f 3f 3f 20 20 20 20 20  |12     ????     |
````

Either find out how to get Unicode out of `wmic` or find a different way to map a adapter index to a name.

**Comment by silverwind** [comment]
Some more info https://stackoverflow.com/a/55312341/808699. Not happy with a extra temp file only to get the correct encoding, so maybe there are better solutions.

**Comment by silverwind** [comment]
Found a workaround, released in 5.0.1.

``` js
> require(".").v6.sync()
{ gateway: '2001:db8::1', interface: '无线上网' }
````

**Comment by WONDERlust91** [comment]
Thanks for your help. Much appreciated!
